src/devices/Ssd13xx/Commands/Ssd1327Commands/SetUnlockDriver.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ssd13xx.Commands.Ssd1327Commands
+{
+    public class SetUnlockDriver : ISsd1327Command
+    {
+        const byte Unlock = 0b_0001_0010;
```

Comment:
should this be coming from the property/constructor? (i.e. bool for lock/unlock)